### DISCUSSION TITLE
REVR dynamic menu placement?

### DISCUSSION THREAD

**Discussion by netriosilver** [praydog/REFramework#277] [General Questions] [answered]
I'm wondering if/when the inventory/menu for the Resident Evil VR ports will be able to pop up in the direction you are facing rather than the direction you were facing when you opened the game. It's really the only gripe I have with the port, and the only thing that really slows down the pace and everything; as a roomscale player using Virtual Desktop, it's quite annoying to do a little spin every time I need to use an item.

**Comment by praydog** [comment, marked as answer]
Download a newer build. This was fixed a while ago.

https://github.com/praydog/REFramework-nightly/releases

**Comment by netriosilver** [comment]
Oh, guess I was using the wrong link or something. That's awesome!
